PopPUNK/utils.py

```diff
@@ -352,7 +353,7 @@ def update_distance_matrices(refList, distMat, queryList = None, query_ref_distM
     # return outputs
     return seqLabels, coreMat, accMat
 
-def assembly_qc(assemblyList, klist, ignoreLengthOutliers, estimated_length):
+def assembly_qc(assemblyList, klist, estimated_length):
```

Comment:
This is duplicated given the new function below, so I would suggest removing the whole thing. It still requires assemblies and reads through them to find their length, which is not ideal – the way below is much more portable. We've dropped mash support now anyway, so should be safe to get rid of this

Reply:
Wasn't sure if mash was definitely gone now - we should probably clear out all the legacy code on its own branch?

Reply:
Gone in 6c95af0.